File: src/main/java/com/github/svart63/controller/WsController.java
package com.github.svart63.controller;

import com.github.svart63.model.RequestMessage;
import org.springframework.messaging.handler.annotation.MessageMapping;
import org.springframework.stereotype.Controller;

import java.util.Collections;
import java.util.List;

@Controller
public class WsController {
    @MessageMapping("/request")
    public List<RequestMessage> messages(String requestId) {
        System.out.println("Request: " + requestId);
        return Collections.emptyList();
    }
}

File: src/main/java/com/github/svart63/controller/WsController.kt
package com.github.svart63.controller

import com.github.svart63.model.RequestMessage
import org.springframework.messaging.handler.annotation.MessageMapping
import org.springframework.stereotype.Controller

@Controller
class WsController {
    @MessageMapping("/request")
    fun messages(requestId: String): List<RequestMessage> {
        println("Request: $requestId")
        return emptyList()
    }
}